src/main/kotlin/com/bookmarks/services/PriceService.kt

```diff
@@ -0,0 +1,37 @@
+package com.bookmarks.services
+
+import com.bookmarks.models.Book
+import com.bookmarks.models.Price
+import com.bookmarks.models.Price.Companion.fromCents
+import com.bookmarks.models.Price.Companion.toCents
+import com.bookmarks.models.User
+import org.springframework.stereotype.Component
+
+@Component
```

Comment:
Why do you need `object` as a `Component`?

Reply:
Fixed.